guides/release/upgrading/index.md

```diff
@@ -1 +1,81 @@
+
+## Routine minor version maintenance upgrades
+
+Let's say you are upgrading an app from `v3.4` to `v3.8`.
+Although you heard there are some new features, your main goal is to keep up with security updates.
+To do this kind of version upgrade, follow the instructions [here in the CLI Guides](https://cli.emberjs.com/release/basic-use/upgrading/).
+That process will make sure that any peer dependencies of the `ember.js` core codebase get upgraded too.
+Automated tools will help you make the right changes to `package.json` and other files.
+The point of a minor version bump is that you shouldn't need to change anything in your app when you upgrade - it should all keep working, whether you choose to adopt new syntaxes and feature, or not.
+
+## Using new features
+
+Once you have upgraded an app's version, some new features may available right of the box.
+Other features may require that you enable them specifically in your app's configuration, since they may change the app's default behavior.
+
+The best way to discover new features is to read the [release blog posts](https://blog.emberjs.com/tags/releases.html).
+If a new feature requires you to opt-in, it's called an optional feature.
+Follow [this guide](../../configuring-ember/optional-features/) to learn which optional features are available in your app's version and how to enable them.
+In many cases, codemods will be available to help you make syntax-related updates.
+
+## Octane upgrade strategy
+
+Octane is Ember's [first Edition](https://emberjs.com/editions/).
+An edition is essentially a collection of features that are available in a minor (non-breaking) release of Ember.
+Although it's possible to do a big-bang refactor to change an app to use all of Octane's features at once, it's really designed for incremental adoption.
+You can make changes at the pace that makes sense for your project or team.
+
+Read on to learn how to begin using Octane's features in an existing application.
+In summary:
+
+1. Follow the [regular upgrade steps](https://cli.emberjs.com/release/basic-use/upgrading/) to update your app to at least version `3.x`. <!-- TODO -->
+2. Run your tests to make sure everything still works as expected
+3. Review the deprecation warnings and make any necessary refactors. Some deprecations will need to be resolved before you can use an Octane feature. Read on to learn what they are.
+4. Go through the [Quick start tutorial](../../getting-started/quick-start) to learn the fundamentals of Octane. You might need to study up on [Native JavaScript Classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/class) too.
+5. Install `@ember/optional-features` in your app, if it is not already in the `devDependencies` of your `package.json`
+6. Turn on optional features one by one, running tests in between to make sure things still work as expected. Read on to learn what an Octane configuration looks like.
```

Comment:
Hm...I wonder if, instead of `Read on to learn...`, we could just link to the `### Optional features in Octane` heading? Is that possible? I think it would be nice to just jump down to the applicable section of the page (but this is just a nice-to-have, not a blocker by any means!)